packages/wine.rb

```diff
@@ -11,18 +11,20 @@ class Wine < Package
   when 'aarch64', 'armv7l', 'x86_64'
     source_url 'https://dl.winehq.org/wine/source/5.x/wine-5.22.tar.xz'
     source_sha256 '09bd06c87c8c974e6ad34507cec875d7217eb56fc09df838d5453e0ebbce4d21'
+    depends_on 'eudev'
+    depends_on 'ffmpeg'
     depends_on 'freetype'
     depends_on 'glib'
-    depends_on 'libgphoto'
-    depends_on 'libxext'
-    depends_on 'eudev'
     depends_on 'gstreamer'
     depends_on 'lcms'
-    depends_on 'openldap'
+    depends_on 'libgphoto'
     depends_on 'libpcap'
-    depends_on 'apulse'
+    depends_on 'libunwind'
+    depends_on 'libx11'
+    depends_on 'libxext'
     depends_on 'mpg123'
-    depends_on 'ffmpeg'
+    depends_on 'openldap'
+    depends_on 'pulseaudio'
```

Comment:
Bump version and generate new binaries?

Reply:
Might as well upgrade to the latest release also.

Reply:
The newest binary I generated doesn't work. So...

Reply:
Just want to get rid of possibility of `apulse` getting installed and overwriting pulse audio.

We really need a central way of preventing overwriting files from existing packages during install and throwing an error... That would solve this issue...

Reply:
Yeah, I'm sure this package requires persistence.  There are a ton of dependencies and it only makes sense to include as much as we can.

Reply:
Updated to new version. Any thoughts on moving the .wine folder location to a XDG_CONFIG_DIR location and also setting that for wine prefix? This does work for me: `WINEPREFIX=$XDG_CONFIG_HOME/.wine wine explorer`

Reply:
Might want to add `depends_on 'xdg_base'` to the list.

Reply:
@uberhacker Can we just add that to core?

Reply:
Maybe but how many packages does it affect?  I'm reluctant to add gui based dependencies in core.  I want to stay away from that bloat potential.

Reply:
`xdg_base` is pretty slim though. Maybe we need a `gui_core`?